cmd/gapir/cc/main.cpp

```diff
@@ -283,6 +287,24 @@ int main(int argc, const char* argv[]) {
     core::Debugger::waitForAttach();
   }
 
+  if (replayArchive) {
```

Comment:
Now that we have a big mode-switch at the start of the replay, I would split this out into 2 methods. One that starts an "archived replay" and one that opens a connection.

It is a bit weird to have a "start the program in an entirely different way" early-out in the middle of main().